templates/shared/pricing/_kubernetes-consulting-add-ons.html

```diff
@@ -1,31 +1,32 @@
-<div class="row u-equal-height">
-  <div class="col-6 p-card">
-    <div class="p-card__header">
-      <h2 class="p-heading--four">Workshop</h2>
+<div class="row">
+  <div class="p-card">
+    <div class="col-12 p-card__header">
+      <h2 class="p-heading--four">AI/ML Add-on</h2>
       <p><span class="p-heading--two">$40,000</span> one off fee</p>
     </div>
-    <p>One additional day on Kubeflow, including  Tensorflow and  JupyterHub, covering everything your business needs to know to have a full on-prem/off-prem AI/ML game plan.</p>
-    <ul class="p-list">
-      <li class="p-list__item is-ticked">On site or remote options</li>
-      <li class="p-list__item is-ticked">Hands-on K8s and Kubeflow</li>
-      <li class="p-list__item is-ticked">Full pipeline view</li>
-      <li class="p-list__item is-ticked">ML / Data science assessment</li>
-    </ul>
-    <p><a href="/kubernetes/contact-us?product=kubernetes-discoverer">Contact us&nbsp;&rsaquo;</a></p>
-  </div>
-  <div class="col-6 p-card">
-    <div class="p-card__header">
-      <h2 class="p-heading--four">ML / Data Science Assessment</h2>
-      <p><span class="p-heading--two">$40,000</span> one off fee</p>
+    <div class="row u-equal-height p-divided">
```

Comment:
Should be `p-divider` (not that it'll actually make a difference with `u-equal-height` already there)

Reply:
Fixed